NAMESPACE

```diff
@@ -41,7 +41,6 @@ S3method(mutate_data,TealDataAbstract)
 S3method(mutate_dataset,TealDataAbstract)
 S3method(mutate_dataset,TealDataset)
 S3method(mutate_dataset,TealDatasetConnector)
-S3method(mutate_join_keys,JoinKeys)
```

Comment:
Why keys can't be mutated anymore? I suggest to keep it as it might be needed again in the near future. For example if we decide to have a function `get_default_cdisc_keys(c("ADSL", "ADTTE", "ADRS"))` which produces a `JoinKeys` object. We would like to be able to modify this object using `mutate_join_keys`.
For example
```r
get_default_cdisc_keys(c("ADSL", "ADTTE", "ADRS")) |>
  mutate_join_keys("ADSL", "ADSL", c("SID", "USID")) |>
  mutate_join_keys("ADTTE", "ADTTE", c("SID", "USID", "PARAMCD")) |>
  mutate_join_keys("ADSL", "ADTTE", c("SID", "USID"))
```

I think we might consider in the future something like this:
````r
get_default_cdisc_keys(c("ADSL", "ADTTE", "ADRS")) |>
  mutate_fk("ADTTE", "ADTTE", c("SID", "USID", "PARAMCD")) |>
  mutate_pk("ADSL", c("SID", "USID")) |>
  mutate_pk("ADTTE",  c("SID", "USID", "PARAMCD")) |>
  mutate_parent("ADTTE", "ADSL")
```